modules/navier_stokes/test/tests/finite_volume/wcns/enthalpy_equation/tests

```diff
@@ -0,0 +1,21 @@
+[Tests]
```

Comment:
I noticed that `enthalpy_equation.i` is not used anywhere. Is that file important? If yes, please add a test for it.

Reply:
added an exodiff test